### DISCUSSION TITLE
Recovering Account after Website Manager AWOL

### DISCUSSION THREAD

**Discussion by SacredLab** [joomla/joomla-cms#39296] [Q&A]
Hey guys,

Fresh to Joomla.
I need to recover some account details, however the email associated with the account (and password) are unknown to they who own the site. The former website manager has disappeared.
I just don't know how to get into the Joomla CMS. Nor do I know how to contact Joomla support, which doesn't seem to exist.

Much appreciated any assistance that can be offered to this end.

Kind regards,
Ryan

**Comment by richard67** [comment]
Check https://docs.joomla.org/How_do_you_recover_or_reset_your_admin_password%3F .
